### PR TITLE
genmypy: 0.3.0-1 in 'noetic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by bonprosoft**
Increasing version of package(s) in repository `genmypy` to `0.3.0-1`:

- upstream repository: https://github.com/rospypi/genmypy.git
- release repository: https://github.com/rospypi/genmypy-release.git
- distro file: `noetic/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

## genmypy

```
* Rename genpyi to genmypy (#35 <https://github.com/rospypi/genmypy/issues/35>)
```

### CODE REVIEW

**Comment by bonprosoft** [comment]
This is another PR to address the review in #29376 and #29377.
Could you review this instead?

As I commented in https://github.com/ros/rosdistro/pull/29376#discussion_r625862878, I renamed my package to `genmypy`, which is less confusing with `genpy`. Please let me know if there is any problem.

File: noetic/distribution.yaml [hunk 1]
```diff
@@ -2229,6 +2229,16 @@ repositories:
       url: https://github.com/ros/genmsg.git
       version: kinetic-devel
     status: maintained
+  genmypy:
+    release:
```
Comment (ivanpauno) [inline noetic/distribution.yaml:2233]: could you add a `doc` entry?

**Review by ivanpauno — APPROVED** [review]

**Comment by ivanpauno** [comment]
> As I commented in #29376 (comment), I renamed my package to genmypy, which is less confusing with genpy. Please let me know if there is any problem.

Naming sounds fine to me :+1:

**Comment by bonprosoft** [comment]
Thank you for your review!
I have added a doc entry for genmypy.

**Pull request merged**